### PR TITLE
feat: add correct wrap [ brakets in chain tagged-optiona

### PR DESCRIPTION
**Pull request by VictorTrumpel**

### CODE REVIEW

File: test/format/value.test.ts [hunk 1]
```diff
@@ -827,10 +827,44 @@ describe('format', function () {
                 {
                     json: {
                         plain: '[[null]]',
-                        html: '<span class="optional">[</span><span class="optional">[</span><span class="yql_null">null</span><span class="optional">]</span><span class="optional">]</span>',
+                        html: '<span class="optional">[[</span><span class="yql_null">null</span><span class="optional">]]</span>',
                     },
                 },
             );
+
+            expect(
```
Comment (Raubzeug) [inline test/format/value.test.ts:835]: let's add test cases to check not null yql.tagged values

**Review by Raubzeug — CHANGES_REQUESTED** [review]

**Review by Raubzeug — APPROVED** [review]
